keras_cv/tests/test_case.py

```diff
@@ -0,0 +1,52 @@
+# Copyright 2023 The KerasCV Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import tensorflow as tf
+from absl.testing import parameterized
+
+from keras_cv.backend import ops
+
+
+class TestCase(tf.test.TestCase, parameterized.TestCase):
+    """Base test case class for KerasCV.
+
+    For now we just extend tf.TestCase and parameterized.TestCase, but this
```

Comment:
Heh, I have this in KerasNLP too, but it's no longer true. We can probably just delete and keep the one liner.

Reply:
yeah this is copied straight from NLP - sorry I didn't give attribution 😓

Reply:
oh all good! we should readily copy back and forth i think

I should just fix this for nlp too